apis/apis.json

```diff
@@ -372,6 +372,15 @@
     }
   },
 
+  {
+    "id": "Google.Cloud.OsLogin.V1Beta",
```

Comment:
It would be nice to have a product URL. I've found https://cloud.google.com/compute/docs/oslogin/rest/ but that's specifically for the REST API, which isn't ideal...